### DISCUSSION TITLE
VSCode Intellisense not working with tailwind 3.1.2?

### DISCUSSION THREAD

**Discussion by stephent** [tailwindlabs/tailwindcss#8591] [Help] [answered]
Watching the 3.1 release [video](https://www.youtube.com/watch?v=nOQyWbPO2Ds), I wanted to try the typescript autocompletions, but I'm not seeing what Adam shows in the video (e.g. no `corePlugins`, no `darkMode` etc.)

![Screen Shot 2022-06-10 at 13 58 43](https://user-images.githubusercontent.com/898090/173142304-7560fc35-8b5b-4579-8b40-d49f4c8deb4a.png)

I already had the [Tailwind CSS Intellisense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) extension installed, but disabling it and reloading didn't make any difference.

Did I miss something else?

**Comment by olivermrose** [comment, marked as answer]
The JSDoc comment needs to be directly above your `module.exports`.

**Reply by stephent** [reply]
That did it - thank you.

**Comment by zackha** [comment]
If you are working with nuxt3, you can try this.

Add the following configuration to your `.vscode/settings.json file`, so that Tailwind directives have proper autocomplete, syntax highlighting, and linting:

```
"tailwindCSS.experimental.configFile": ".nuxt/tailwind.config.cjs"
"files.associations": {
    "*.css": "tailwindcss"
}
```

detailed: [link](https://tailwindcss.nuxt.dev/tailwind/editor-support)
